drivers/virtio/Kconfig

```diff
@@ -0,0 +1,47 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+menuconfig DRIVERS_VIRTIO
+	bool "Virtio Device Support"
+	depends on !OPENAMP
+	default n
+	---help---
+
+if DRIVERS_VIRTIO
+menuconfig DRIVERS_VIRTIO_MMIO_NUM
+	int "The number of virtio mmio devices"
+	default 1
+	---help---
+
+menuconfig DRIVERS_VIRTIO_MMIO_BASE
+	hex "Virtio-mmio base address"
+	default 0x0
+	---help---
+
+menuconfig DRIVERS_VIRTIO_MMIO_REGSIZE
+	hex "Virtio-mmio register size"
+	default 0x0
+	---help---
+
+menuconfig DRIVERS_VIRTIO_MMIO_IRQ
+	int "Virtio-mmio irq number"
+	default 0
+	---help---
+
+menuconfig DRIVERS_VIRTIO_NET
```

Comment:
DRIVERS_VIRTIO_MMIO_NET and depends on DRIVERS_VIRTIO_MMIO_NUM > 0

Reply:
done